unified_targets/ configs/HGLRCF405AS.config

```diff
@@ -0,0 +1,127 @@
+# Betaflight / STM32F405 (S405) 4.0.4 Jun 30 2019 / 14:58:15 (f3a95efa3) MSP API: 1.41
```

Comment:
I don't know what you are trying to do, but now there **is** a space in the directory name. Please fix.